polygon/sync/execution_client.go

```diff
@@ -0,0 +1,29 @@
+package sync
+
+import (
+	"github.com/ledgerwatch/erigon/cl/phase1/execution_client"
+	"github.com/ledgerwatch/erigon/core/types"
+)
+
+type ExecutionClient interface {
+	InsertBlocks(headers []*types.Header) error
```

Comment:
should this be InsertHeaders? or is the plan for it different?

Reply:
The execution API accepts full blocks at the moment: 
https://github.com/ledgerwatch/interfaces/blob/master/execution/execution.proto#L174
we'll need to bring the bodies as well.